rust/agama-server/src/network/model.rs

```diff
@@ -459,7 +514,85 @@ impl Default for Connection {
     }
 }
 
-#[derive(Default, Debug, PartialEq, Clone)]
+impl TryFrom<NetworkConnection> for Connection {
```

Comment:
Finding better names for `NetworkConnection` and friends is starting to be urgent. By now, I would import them as `settings::NetworkConnection` to make clear where they come from.